BUILD.gn

```diff
@@ -226,7 +226,6 @@ if (!is_mac && !is_android && !is_ios) {
           deps = [ "//brave/components/brave_rewards/resources/extension:locales_$locale" ]
 
           locale = string_replace(locale, "-", "_")
-          locale = string_replace(locale, "nb", "no")
```

Comment:
why was this here in the first place?
cc: @simonhong

Reply:
honestly I can't remember why I did like that - Maybe I also did copy & paste from other places..

Reply:
@bbondy thoughts on this? https://github.com/brave/brave-core/commit/cdbfca0d224f005bc16f974a7368d84d10dfca58#diff-71295ce804b64120ddc313f20e26c8adR135

Reply:
It used to be that way in Chromium as well, my guess is they changed it on one of the rebases and we didn't update our code accordingly.